xcode/Subconscious/Shared/Components/OmniboxView.swift

```diff
@@ -9,9 +9,11 @@ import SwiftUI
 
 struct OmniboxView: View {
     @Environment(\.colorScheme) var colorScheme
+    @State private var phase = Double.pi / 8
```

Comment:
What is the value of making this a mutable state vs an immutable property?

If mutability needed, would be nice to have a little comment detailing impl.

Non-blocking.